mesonbuild/scripts/gettext.py

```diff
@@ -31,7 +31,7 @@
 parser.add_argument('--msginit', default='msginit')
 parser.add_argument('--extra-args', default='')
 
-def read_linguas(src_sub: str) -> T.List[str]:
+def read_linguas(src_sub: str) -> T.Tuple[T.List[str], T.Optional[str]]:
```

Comment:
Would be simpler to pass state as arg and do state.interpreter.add_build_def_file(def_file) here. Or even move that function as method on state object since it's used by multiple modules?

Reply:
Hum, no, this is gettext script and not the module.

Reply:
It's also used by scripts/gettext.py though... if you do not specify `i18n.gettext(..., languages: ['list', 'of', 'languages']` then we invoke `meson --internal gettext update_po` without a list of languages, and it handles those on its own.